core/shared/src/main/scala/spire/std/bigDecimal.scala

```diff
@@ -77,7 +78,7 @@ trait BigDecimalIsNRoot extends NRoot[BigDecimal] {
   def nroot(a: BigDecimal, k: Int): BigDecimal = {
     if (a.mc.getPrecision <= 0)
       throw new ArithmeticException("Cannot find the nroot of a BigDecimal with unlimited precision.")
-    NRoot.nroot(a, k, a.mc)
+    spire.math.nroot(a, k, a.mc)
```

Comment:
Ah, cool, so we should definitely remove the existing code in `NRoot`. Or, I'd suggest we maybe use `Algebraic.nroot(a.bigDecimal, k, a.mc)` instead and delete the the shifting n-root algo entirely.